### ISSUE TITLE
new warnings from R 3.6.0

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1535]
The build-test now produces quite a few warnings like this. Since I have my system set up to convert warnings to errors, I plan to see how to get rid of these. I am pretty sure this is part of what I mentioned in #

```R
test_deprecation.R:26: warning: defunct
`addColumn(ctd, ctd[["pressure"]], "pressure")` generated a condition with class defunctError/error/condition.
It is less fragile to test custom conditions with `class`
```

**Comment by dankelley** [comment]
Hm. I've done some reading, and do not know how to test for error messages relating to deprecation. Possibly a new version of `testthat::expect_error()` will handle this. In the meantime, I will simply remove `tests/testthat/test_deprecation.R`, because (a) I want oce to pass build tests and (b) all these tests do is to ensure that error messages have not changed, which is not an important thing to test.

Actually, I think it's time to **remove** those defunct features, according to the usual rules, because the version on CRAN has them marked as "defunct".  Maybe we can have a developer vote on this, with -1 meaning "keep the defunct functions for another CRAN cycle" and +1 meaning "remove the defunct functions". Thoughts, @richardsc? (We could delay for an f2f meeting, if you like, but this is the kind of thing I forget.) I will leave this issue open until Wednesday morning and decide based on votes (or "wait for f2f") by then.

**DK vote: +0.5**

**Comment by dankelley** [comment]
fixed in "develop"; see also #1533.
